fosspay/app.py

```diff
@@ -89,3 +93,51 @@ def inject():
         'str': str,
         'int': int
     }
+
+# STRIPE CODE
+@app.route('/support/confirm_payment', methods=['POST'])
+def make_payment_intent():
+    # Creates a new PaymentIntent with items from the cart.
+    data = json.loads(request.data)
+    try:
+        amount = 500
+        currency = data['currency']
+        # START TODO refactor this code
+        payment_methods = _cfg('payment-methods').split(', ')
+        while("" in payment_methods) : 
+            payment_methods.remove("") 
+        if len(payment_methods) == 0 :
+            payment_methods = ['card']
+        # END TODO 
+        payment_intent = stripe.PaymentIntent.create(
+            payment_method=data['payment_method_id'],
+            amount=1099,
+            currency='usd',
+            confirmation_method='manual',
+            confirm=True,
+        )
+        return jsonify({'paymentIntent': payment_intent})
+        # return generate_payment_response(payment_intent)
```

Comment:
is this going to be used in the future? If so, can it be documented with a comment or removed otherwise?